nuxt-shopify-accounts/api/package-lock.json

```diff
@@ -1,8 +1,1076 @@
 {
```

Comment:
I wonder if the changes in the `package-lock.json`s in this PR are because of installing with a non-LTS node version?

If so, we might want to add `engines` to `package.json` to enforce LTS, and reinstall to update the `package-locks.json`s.